services/app-web/src/components/FileUpload/FileProcessor/FileProcessor.tsx

```diff
@@ -4,6 +4,7 @@ import { FileRow } from '../FileRow/FileRow'
 import { FileListItem } from '../FileListItem/FileListItem'
 
 import styles from '../FileUpload.module.scss'
+import { DocumentCategoryType } from '../../../common-code/domain-models'
```

Comment:
Technically this might actually want to be imported from the generated GraphQL code instead of domain models, one of the reasons I'm excited for us to axe the GraphQL version of the DraftSubmission someday. It's fine as is I mostly just point this out in case at some point this breaks and knowing this might help.